src/plots/Volume/avtDefaultRenderer.h

```diff
@@ -46,17 +48,21 @@ class avtDefaultRenderer : public avtVolumeRendererImplementation
     virtual void               Render(const RenderProperties &props,
                                       const VolumeData &volume);
 
-    vtkColorTransferFunction  *transFunc;
-    vtkPiecewiseFunction      *opacity;
-    vtkVolume                 *curVolume;
-    vtkImageData              *imageToRender;
-    vtkVolumeProperty         *volumeProp;
-    vtkSmartVolumeMapper      *mapper;
+    vtkColorTransferFunction  *transFunc    {nullptr};
+    vtkPiecewiseFunction      *opacity      {nullptr};
```

Comment:
Why initializers here, when they are initialized in the constructor?

Reply:
Fell swoop of initializing everything in the header regardless of what happens in the C file. Selective inits in the header are confusing so do them all.